### ISSUE TITLE
Theme Check flags "wordpress" identifier as REQUIRED — false positive, do not "fix" by renaming

### ISSUE THREAD

**Issue by lihsaa591** [Otto42/theme-check#505]
**Context:** WordPress.org's automated Theme Check scan on the release build reported:

```
REQUIRED: Found incorrect spelling or capitalization of WordPress in the file

Line 273:    'wordpress'      => [
Line 274:        'value' => 'wordpress',

```
**Why this is a false positive, not a bug:**

```
'wordpress' => [
    'value' => 'wordpress',
    'label' => 'WordPress',   // ← already correctly capitalized
],
```

The display text is already correct. The key/value are internal, code-level identifiers (used consistently the same way for every other social network), almost certainly feeding into things like a CSS/icon class or a stored customizer setting.

**Why we should not "fix" this by renaming the key/value:**

- Breaks consistency with every other entry in the array
- Risks breaking icon rendering, if value feeds a fa-{value} class construction
- Risks breaking backward compatibility for any site that already saved 'wordpress' as a selected value in its database

Theme Check's scanner does a blind text search for "wordpress" and doesn't understand code context — this is a well-known false-positive pattern for icon/data lookup tables. Renaming a working, consistently-used identifier just to silence a linter that's wrong here would trade real functionality/compatibility risk for a cosmetic pass on an automated check.

**Recommended resolution:** Leave the code as-is. The REQUIRED flag routes the release to WP.org's manual review queue rather than blocking it outright — a human reviewer will see the label is already correctly capitalized and can clear it. Document this decision here so it isn't "fixed" incorrectly in a future pass without this context.
